

func ret_err_func(int32 a)
	-> int32 b, int16 b { 3->b }
